src/main/kotlin/minesweeper/domain/land/Land.kt

```diff
@@ -28,7 +26,7 @@ class Land private constructor(val spots: List<Spot>, val land: LandSize) {
                     val x = index % land.width
                     when (mines.contains(y * land.width + x)) {
                         true -> MineSpot(Point(x, y))
-                        false -> DefaultSpot(Point(x, y))
+                        false -> DefaultSpot(Point(x, y), MineData(land, mines))
```

Comment:
코틀린에서는 2가지 조건인 경우 if-else를 사용하는 것을 권장합니다.
> Prefer using if for binary conditions instead of when. 

ref. https://kotlinlang.org/docs/coding-conventions.html#if-versus-when

Reply:
if else 로 수정하였어요